packages/components/Molecules/Form/index.vue

```diff
@@ -71,6 +71,17 @@ export default {
   },
   mounted () {
     RplFormEventBus.$on('clearform', this.clearForm)
+    VueFormGenerator.validators.rplWordCount = function (value, field) {
```

Comment:
Can you please add a `TODO` comment for this, we need to move custom validators into separated files in future. We will add unit test in that time.